src/test/java/ui/ConfigProperties.java

```diff
@@ -0,0 +1,31 @@
+package ui;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    protected static Properties properties;
```

Comment:
Is there a reason to have this variable `protected` while there's a `getProperty` method?